### ISSUE TITLE
Dropdown accessibility

### ISSUE THREAD

**Issue by sahilamoli** [react-toolbox/react-toolbox#1429]
I have a dropdown with a custom template and it seems like tabbing into it does not work. When there is no template, it seems to work.

Additionally, the dropdown component doesn't let you use arrow keys to select different options.

**Comment by sahilamoli** [comment]
Hi, is this project not being maintained anymore? Or am I wrong in assuming that this should be a pretty high-priority item for any scalable component library?

**Comment by tobiasandersen** [comment]
Yes it is still being maintained (primarily by a single person who's had trouble finding extra time to work a lot on it in the last couple of weeks). You're very welcome to fix it yourself and create a PR from it!

**Comment by sahilamoli** [comment]
Thanks for letting me know!

**Comment by sahilamoli** [comment]
Hey @tobiasandersen, I created a pull request to add support for this -- https://github.com/react-toolbox/react-toolbox/pull/1718

It's my first pull request to react-toolbox so let me know if I need to change anything.

Thanks!
